cmd/cloud/setup.go

```diff
@@ -406,7 +417,11 @@ func checkAPIToken(isDeploymentFile bool, coreClient astrocore.CoreClient) (bool
 	org := orgs[0]
 	orgProduct := fmt.Sprintf("%s", *org.Product) //nolint
 
-	err = c.SetContextKey("workspace", workspaceID) // c.Workspace
+	if workspaceID == "" {
```

Comment:
shouldn't this be `wsID` or am I missing anything?

Reply:
you are right...missed replacing that